applications/mlflow/chart/mlflow/templates/minio-tenant.yaml

```diff
@@ -0,0 +1,203 @@
+{{- if .Values.minio.enabled }}
+{{- with .Values.minio.tenant }}
+apiVersion: minio.min.io/v2
+kind: Tenant
+metadata:
+  name: {{ .name }}
+  labels:
+    app: minio
+  {{- if dig "metrics" "enabled" false . }}
+  ## Annotations for MinIO Tenant Pods
+  annotations:
+    prometheus.io/path: /minio/v2/metrics/cluster
+    prometheus.io/port: {{ dig "metrics" "port" 9000 . | quote }}
+    prometheus.io/scrape: "true"
+    prometheus.io/scheme: {{ dig "metrics" "protocol" "http" . | quote }}
+  {{- end }}
```

Comment:
I notice there aren't global settings for lables/annotations here. Is that for future implementation or intentional?

Reply:
this isn't consistent everywhere at the moment. i'll create a story to make sure we have global labels and annotations, and then also annotations at the object level for most(probably everything excluding configmap/secret)